experimental/packages/opentelemetry-sdk-metrics-base/src/aggregator/Histogram.ts

```diff
@@ -141,14 +141,14 @@ export class HistogramAggregator implements Aggregator<HistogramAccumulation> {
     startTime: HrTime,
     endTime: HrTime): Maybe<HistogramMetricData> {
     return {
-      instrumentDescriptor: metricDescriptor,
-      pointDataType: PointDataType.HISTOGRAM,
-      pointData: accumulationByAttributes.map(([attributes, accumulation]) => {
+      descriptor: metricDescriptor,
```

Comment:
Nit: `instrumentDescriptor` was named to `descriptor` in `LastValue` and `Sum`, should perhaps change `metricDescriptor` to just descriptor as well to keep it consistent?

Reply:
Thank you for pointing this out! Fixed.